src/perishable.ts

```diff
@@ -5,6 +5,7 @@ class PerishableNode<T> implements Handle<T> {
     _callback: () => any;
     _prev: PerishableNode<T>;
     _next: PerishableNode<T>;
+    _is_released: boolean;
```

Comment:
It's not publicly exported and is only used by the Perishable class. If it was private, the perishable class would not be able to do as much with it.